Day 6 of python
Python try ... exception
0x05-python-exceptions
